Sources/PodToBUILD/GlobUtils.swift

```diff
@@ -499,6 +499,21 @@ public func podGlob(pattern: String) -> [String] {
     return Glob(pattern: pattern, behavior: GlobBehaviorBashV4).paths
 }
 
+// Expands a set of globs with the semantics of pod `source_file` globs.
```

Comment:
Note: This is just the functionality that was in `Sources/RepoToolsCore/RepoActions.swift`, but pulled out to a function for easy reuse.